cc3d/core/Configuration/SettingUtils.py

```diff
@@ -212,3 +231,51 @@ def synchronize_global_and_default_settings(default_settings, global_settings, g
 
 def synchronizeGlobalAndDefaultSettings(default_settings, global_settings, global_settings_path):
     return synchronize_global_and_default_settings(default_settings, global_settings, global_settings_path)
+
+
+def _handle_settings_value_xml(_val: str):
+    """
+    Handles data intended for sqlite but used in python types
+    """
+    if _val == 'True':
+        return '1'
+    elif _val == 'False':
+        return '0'
+    return _val
+
+
+def default_settings_dict_xml() -> Dict[str, Any]:
+    """
+    Returns default data dictionary
+    """
+    fp = _default_setting_path_xml()
+    if not os.path.isfile(fp):
+        warnings.warn(f'Could not located default settings data ({fp})')
+        return {}
+
+    data_root = ElementTree.parse(fp).getroot()
+    sz_util = SerializerUtil()
+
+    def _import_data(_el):
+        _val_text = _handle_settings_value_xml(_el.text)
+        _type_fcn = sz_util.type_2_deserializer_dict[_el.attrib['Type']]
+        return _el.attrib['Name'], _type_fcn(_val_text)
+
+    result = {}
+    for el in data_root.findall('Settings'):
+        for el_e in el.findall('e'):
+            if el_e.attrib['Type'] == 'dict':
+                result_e = {}
+                for el_e_e in el_e.findall('e'):
+                    try:
+                        result_e.__setitem__(*_import_data(el_e_e))
+                    except (KeyError, TypeError):
+                        pass
+                result[el_e.attrib['Name']] = result_e
+            else:
+                try:
+                    result.__setitem__(*_import_data(el_e))
+                except (KeyError, TypeError):
+                    pass
```

Comment:
Should these `except` blocks print something like, "The setting for ___ could not be read"?

Reply:
Agreed. Addressed in https://github.com/CompuCell3D/CompuCell3D/pull/710/commits/c592f764ecd9cdd9bf186665fe93830480a1090c.